spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -11,6 +11,40 @@ def create_generic_work
     generate 'hyrax:work GenericWork'
   end
 
+  def create_nested_work
```

Comment:
If we add a new work type, can we get rid of AtlasWork?  It's only purpose was to be a second work type, which is now served by NestedWork.  Having only two work types generated will reduce the build time.